src/sestest/GenerateCommandLine.cs

```diff
@@ -34,5 +34,8 @@ public sealed class GenerateCommandLine : CommandLineBase
 
         [Option("audience", HelpText = "Audience to which tokens will be addressed (optional; defaults to 'https://batch.azure.test/software-entitlement').")]
         public string Audience { get; set; }
+
+        [Option("issuer", HelpText = "Issuer by whom the tokens are created (optional; defaults to 'http://batch.azure.test/software-entitlement').")]
```

Comment:
So as an ISV doing dev against this SDK with a view to running against the real SES - what should I use here?  Should I accept the default?  When should I override the default?  If I override it what do I need to do in other parts of my test system?